packages/caip/src/adapters/coingecko/utils.test.ts

```diff
@@ -69,6 +70,13 @@ const makePolygonMockCoingeckoResponse = () => ({
   platforms: {},
 })
 
+const makeGnosisnMockCoingeckoResponse = () => ({
+  id: 'gnosis',
+  symbol: 'gno',
```

Comment:
Shouldn't this be mocking XDAI? GNO is the governance token, not sure that's what we want to test here?